$NetBSD: patch-ae,v 1.3 2005/09/27 11:25:37 dsainty Exp $

This program has been severely patched, and is now a "NetBSD PIC programmer".
Make the title reflect this.  Correct the incorrect version number too.

--- picprg.h	2005-09-13 03:09:58.000000000 +1200
+++ picprg.h	2005-09-13 03:08:21.000000000 +1200
@@ -11,7 +11,7 @@
 #define flushall() fflush(NULL)
 
 
-#define VERSION " Linux PIC programmer     v2.3c           Copyright 1994-1998 by Brian C. Lane "
+#define VERSION " NetBSD PIC programmer     v2.3d           Copyright 1994-1998 by Brian C. Lane "
 
 #define	PIC16C84	0
 
